README.md

```diff
@@ -53,3 +53,6 @@ The email address of a user account that has access to the drive folder and will
 
 #### `override` (Optional):
 If set true, delete files with the same name before uploading.
+
+#### `depth` (Optional):
```

Comment:
You have not set up `depth` as a parameter, `MAX_DEPTH` is hardcoded in `getUploadFolderId(...)`. We can either:

1. add this as a parameter, better named `max_depth`, and add to parameter awareness in `main()` via `actions.getInput(...)` or the logging wrapper `getInputAndDebug(...)`, or
2. remove from the README, replacing with a note of what the hardcoded maximum folder depth is

As a more minimal change, that I do not think is controversial, I would opt for (2)

Reply:
Option 2 seems like a good choice for now.
The only getInput() returns a string, seems like testing parsing a number could be another task:


We could potentially do something along the lines of: 
```
function getNumberInput(name) {
  const input = actions.getInput(name)
  if (input === ''){
    return // return undefined
  }
  cont value = Number(input)
  if (Number.isNaN(value)) {
    throw new Error(`input ${name} could not be parsed as number, received value: ${input}`);
  }
  return value
}

const maxDepth = getNumberInput('max_depth') ?? 15
```

Reply:
Okay, we can leave that as a separate tasks we may wish to do later. Focusing on resolving the security concerns, we can go with 2 then. I believe John said he was out on leave tomorrow (Friday) so I can take this change on.